src/API/Management/GenericResource.php

```diff
@@ -75,15 +75,19 @@ protected function normalizePagination(array $params, $page = null, $per_page =
      *
      * @return array
      */
-    protected function normalizeIncludeTotals(array $params, $include_totals = false)
+    protected function normalizeIncludeTotals(array $params, $include_totals = null)
     {
         // User parameter include_totals if params does not have the key.
         if (! isset( $params['include_totals'] )) {
             $params['include_totals'] = $include_totals;
         }
 
-        // Make sure we have a boolean.
-        $params['include_totals'] = filter_var( $params['include_totals'], FILTER_VALIDATE_BOOLEAN );
+        // Unset if NULL, or else make sure we have a boolean.
+        if (is_null($params['include_totals'])) {
```

Comment:
You can remove this check. I'll submit a PR to exclude any parameters set to `null` in the parameter builder. That way we can avoid these checks in other "normalize params" methods we might add in the future. If you add `FILTER_NULL_ON_FAILURE` as a third param, then we can handle all irrational arguments the same way.